guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -79,6 +88,11 @@ ifdef::katello,orcharhino,satellite[]
 include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+2]
 endif::[]
 
+ifdef::foreman-el,katello,satellite[]
+// Upgrading Project from Enterprise Linux 7 to Enterprise Linux 8 using Leapp
+include::common/modules/proc_upgrading-project-in-place-using-leapp.adoc[leveloffset=+1]
```

Comment:
What about foreman-deb and orcharhino?
cc @maximiliankolb

Reply:
foreman-deb can't be upgraded via Leapp